HeartbeatLogging/Sources/HeartbeatLoggingTestUtils.swift

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import XCTest
+
+@objc(FIRHeartbeatLoggingTestUtils)
+@objcMembers
+public class HeartbeatLoggingTestUtils: NSObject {
```

Comment:
When integrating with FirebaseCore, I realized during tests that it'd be helpful to have the `assertEqualPayloadStrings` custom assertion so i created this file to add it to the `HeartbeatLogging` module.